docs/integration/infra.md

```diff
@@ -0,0 +1,276 @@
+title: Infrastructure Monitoring Integration
+description: Infrastructure performance monitoring - metrics, events, processes, packages, log collection and parsing for bare-metal servers, virtual machines and containers with support for Docker, containerd, k8s, swarm, nomad, ecs, eks, aks, gke.
+
+The Sematext Infra Integration uses [Sematext Agent](../agents/sematext-agent) to collect metrics, events, processes, packages for bare-metal servers, virtual machines and containers. Container monitoring is supported for Docker, containerd, k8s, Swarm, Nomad, ECS, EKS, AKS, GKE and more.
+
+We at Sematext aim to save you time and effort by giving you a strong starting point for monitoring your infrastrcture. You will **not** have to:
+
+- figure out which metrics to collect and which ones to ignore
+- give metrics meaningful labels
+- hunt for metric descriptions in the docs so that you know what each of them actually shows
+- build charts to group metrics that you really want on the same charts, not N separate charts
+- figure out, for each metric, which aggregation to use (min? max? avg? something else?)
+- build dashboards to combine charts with metrics you typically want to see together
+- set up basic alert rules
+
+We set it up for you, out-of-the-box!
```

Comment:
We set it up for you, out-of-the-box! => Sematext provides all that and more out of the box.

Reply:
I'll adjust all this, but just FYI, this and most of the other stuff was present before under docker integration doc, I just copy-pasted that stuff directly into new infra integration doc. So this is re-reviewing and fixing something that was already public.

Reply:
Yeah.  But some stuff is outdated, some stuff can simply be improved, etc.